src/slick.dataview.ts

```diff
@@ -429,7 +429,11 @@ export class SlickDataView<TData extends SlickDataItem = any> implements CustomD
       gi.compiledAccumulators = [];
       let idx = gi.aggregators.length;
       while (idx--) {
-        gi.compiledAccumulators[idx] = this.compileAccumulatorLoop(gi.aggregators[idx]);
+        if(this._options.useCSPSafeFilter){
+          gi.compiledAccumulators[idx] = this.compileAccumulatorLoopCSPSafe(gi.aggregators[idx]);
+        }else {
```

Comment:
small lint nitpick, can you add space after curly brace and also add a space after bracket on the if condition.